packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -168,7 +171,8 @@ class IconButton extends StatelessWidget {
     return new InkResponse(
       onTap: onPressed,
       child: result,
-      radius: math.max(size, Material.defaultSplashRadius),
+      radius: math.max(48.0, iconSize + math.min(padding.horizontal, padding.vertical))
+          * 0.55, // x 0.5 for diameter -> radius and + 10% overflow so it doesn't look too square.
```

Comment:
Material.defaultSplashRadius is a static number right? I didn't find anything specific in the material guidelines. Though looking at images in the website and other material apps, a 70px splash on a 48px button with a 24px icon looks too big. 

Let's look at it together tomorrow.